test/itg/1.7/install_prestashop.js

```diff
@@ -137,6 +137,11 @@ function runScenario() {
                     .click(this.selector.InstallationWizardPage.test_conection_button)
                     .waitForExist(this.selector.InstallationWizardPage.dbResultCheck_green_block, 300000)
                     .call(done);
+                this.client
+                    .waitForExist(this.selector.InstallationWizardPage.dbResultCheck_red_block, 3000)
+                    .getText(this.selector.InstallationWizardPage.dbResultCheck_red_block).then(function (text) {
+                        console.error("Could not confirm database credentials.", text);
+                    });
```

Comment:
@fatmaBouchekoua do you know how to stop the test? We can't go further if the database is not validated, and PrestaShop is not installed.

Reply:
Try ```this.client.end()``` to end the session and close the browser

Reply:
Unfortunately the script was still running :(